docs/api/v2/node.md

```diff
@@ -53,11 +53,11 @@ method publish*(w: WakuNode, topic: Topic, message: WakuMessage)
   ##
   ## Status: Implemented.
 
-method query*(w: WakuNode, query: HistoryQuery): HistoryResponse
-  ## Queries for historical messages.
+method query*(w: WakuNode, query: HistoryQuery, handler: QueryHandlerFunc) {.async, gcsafe.} =
+  ## Queries known nodes for historical messages. Triggers the handler whenever a response is received.
```

Comment:
node*s* or node? If you connected to 10 nodes how is it supposed to work? When is it done? This is why the spec should be clarified here https://github.com/vacp2p/specs/issues/187

I would personally expect this to start with a single node (up to implementation how that is chosen) for now and then look at expanding capabilities later.

Reply:
We currently use a single node. I added the todo on how nodes get picked and believe this would fall into that.